routing/index_graph_serialization.hpp

```diff
@@ -323,7 +323,7 @@ class IndexGraphSerializer final
     Joint::Id GetCount() const { return m_count; }
 
   private:
-    static uint32_t constexpr kEmptyEntry = numeric_limits<uint32_t>::max();
+    static uint32_t constexpr kEmptyEntry = std::numeric_limits<uint32_t>::max();
```

Comment:
static auto constexpr? and below

Reply:
first yes, second no
I don't have to look recursively to know the type

Reply:
they're so close to each other...
could you please do ```numeric_limits<uint32_t>::max() - 1``` then?

Reply:
done